framework/log/Dispatcher.php

```diff
@@ -113,7 +113,7 @@ public function init()
     public function getLogger()
     {
         if ($this->_logger === null) {
-            $this->setLogger(Yii::getLogger());
+            $this->setLogger('yii\log\Logger');
```

Comment:
@TerraSkye correct, but that decision has been made after @sergeymakinen commented ;)